DEMOQA/cypress/tests/login.js

```diff
@@ -0,0 +1,204 @@
+describe('login', () => {
+
+    const testData = require('../fixtures/testData.json')[0]
+    beforeEach(() => {
+        cy.visit("https://demoqa.com/automation-practice-form")
+    });
+    it('submit successfully with valid input', () => {
+        cy.fixture('testData').then((data) => {
+            cy.get("#firstName").type(data.firstName)
+            cy.get("#lastName").type(data.lastName)
+            cy.get('#userEmail').type(data.userEmail)
+            cy.get("label[for='gender-radio-1']").click();
+            cy.get("#userNumber").type(data.userNumber)
+            cy.get('#dateOfBirthInput').click()
+            cy.get('.react-datepicker__month-select').select('January')
+            cy.get('.react-datepicker__year-select').select('2001')
+            cy.get('[role = "option"]').contains('15').click()
+            cy.get('#subjectsContainer').type('m{enter}')
+            cy.get('#uploadPicture').selectFile('cypress/fixtures/ok.png')
+            cy.get('label[for="hobbies-checkbox-1"]').click();
```

Comment:
For checkbox, try do implement with 2 or more checkboxes checked, store data in an array and loop here